ftp-ballerina/tests/secure_listener_endpoint_test.bal

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/lang.runtime as runtime;
+import ballerina/log;
+import ballerina/test;
+
+int secureAddedFileCount = 0;
+int secureDeletedFileCount = 0;
+boolean secureWatchEventReceived = false;
+
+listener Listener secureRemoteServer = new({
+    protocol: SFTP,
+    host: "127.0.0.1",
+    auth: {
+        basicAuth: {
+            username: "wso2",
+            password: "wso2123"
+        },
+        privateKey: {
+            path: "tests/resources/sftp.private.key",
+            password: "changeit"
+        }
+    },
+    port: 21213,
+    path: "",
```

Comment:
Do we need this property here?

Reply:
Yes this will mount the file system to the virtual root.